website/docs/docs-section/how-does-it-work/document-integrity.md

```diff
@@ -1,9 +1,10 @@
 ---
+id: document-integrity
 title: Document Integrity
 sidebar_label: Document Integrity
 ---
 
-OpenAttestation ensures that the content of the document has not been modified since the document was created, with the exception of data removed using the built-in [obfuscation mechanism](/docs/developer-section/libraries/remote-files/open-attestation#obfuscating-data). Let's explore how it works.
+OpenAttestation ensures that the content of the document has not been modified since the document was created, with the exception of data removed using the built-in [obfuscation mechanism](/docs/developer-section/libraries/remote-files/open-attestation#obfuscating-data). This article describes how it works.
```

Comment:
We can remove the last line entirely
```suggestion
OpenAttestation ensures that the content of the document has not been modified since the document was created, with the exception of data removed using the built-in [obfuscation mechanism](/docs/developer-section/libraries/remote-files/open-attestation#obfuscating-data). 
```

Reply:
deleted